it/controllers/updateRegisteredDetails/ChangeRegisteredDetailsControllerISpec.scala

```diff
@@ -0,0 +1,232 @@
+package controllers.updateRegisteredDetails
+
+import controllers.ControllerITTestHelper
+import models.SelectChange.UpdateRegisteredDetails
+import models.updateRegisteredDetails.ChangeRegisteredDetails
+import org.jsoup.Jsoup
+import org.scalatest.matchers.must.Matchers.convertToAnyMustWrapper
+import pages.updateRegisteredDetails.ChangeRegisteredDetailsPage
+import play.api.http.HeaderNames
+import play.api.libs.json.Json
+import play.api.test.WsTestClient
+
+class ChangeRegisteredDetailsControllerISpec extends ControllerITTestHelper {
+
+  val normalRoutePath = "/"
+
+  "GET " + normalRoutePath - {
```

Comment:
can we check which checkboxes appear based on the "voluntary" flag for the GET AND POST(400) in the it tests